base/common/src/main/java/com/netscape/certsrv/request/RequestId.java

```diff
@@ -28,10 +42,14 @@
  *
  * @version $Revision$ $Date$
  */
+@XmlRootElement(name = "authority")
+@XmlAccessorType(XmlAccessType.FIELD)
+@JsonInclude(Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown=true)
 public class RequestId implements Serializable {
 
     private static final long serialVersionUID = -5184710368124269481L;
-    protected BigInteger value;
+    public BigInteger value;
```

Comment:
For a class that contains only a single field or a simple value, I think we can use `@JsonValue` like this:
https://github.com/dogtagpki/pki/blob/master/base/common/src/main/java/com/netscape/certsrv/dbs/certdb/CertId.java#L109

Reply:
OK, good to know, done.